### ISSUE TITLE
How to execute LisGAN.

### ISSUE THREAD

**Issue by naumanbashir70** [lijin118/LisGAN#5]
Hi, @lijin118 

I am trying to run your program but it is giving me error continuously. I have run LisGAN.py but is asked for dataset, I have downloaded dataset and properly set it path in your code. Now, it is giving h5py error. Does it need conversion from .MAT file to .hdf5?

Thanks

**Comment by HuangYL-0619** [comment]
https://github.com/naumanbashir70 Hello! when I run this program,I find it can't run succeedly,Do you have such problem like
File "F:/PythonProject/try0/lisgan.py", line 262, in
sample()
File "F:/PythonProject/try0/lisgan.py", line 135, in sample
input_att.copy_(batch_att)
RuntimeError: The size of tensor a (1024) must match the size of tensor b (85) at non-singleton dimension 1

**Comment by naumanbashir70** [comment]
@yuran1024 
Which dataset you are using?

**Comment by HuangYL-0619** [comment]
@naumanbashir70 I downloaded datasets through its link.When I run this program using AWA1、APY、SUN、or CUB,it has the same problem.The problem happened in function “sample” in lisgan.py.Do you have some advice?

**Comment by naumanbashir70** [comment]
Are you running the LigGAN.py directory or with the parameters.
I was receiving this error.
KeyError : Could not found train_loc . as the dataset was changed.

**Comment by HuangYL-0619** [comment]
@naumanbashir70 I have solved this problem.The name in the file doesn't match the name in the code.You can read "README" and find its name are "att" and "original_att".

**Comment by naumanbashir70** [comment]
Your code is running now?

**Comment by HuangYL-0619** [comment]
@naumanbashir70 No,I have a new problem.The tenosr doesn't match.I  don't know how to debug it.Do you have some advice?

**Comment by naumanbashir70** [comment]
Send me the complete screenshot of error.

**Comment by HuangYL-0619** [comment]
@naumanbashir70    
![image](https://user-images.githubusercontent.com/62084577/76511331-0594e880-648e-11ea-8952-5215ab388d2a.png)

**Comment by naumanbashir70** [comment]
Changes the att size in lisgan.py according to your dataset.

**Comment by naumanbashir70** [comment]
Send me your 
lisga.py and util.py
Thanks

**Comment by HuangYL-0619** [comment]
@naumanbashir70 
[try0.zip](https://github.com/lijin118/LisGAN/files/4327232/try0.zip)

Thank you for your help!

**Comment by naumanbashir70** [comment]
Try this
[lisgan.zip](https://github.com/lijin118/LisGAN/files/4328365/lisgan.zip)

**Comment by HuangYL-0619** [comment]
@naumanbashir70  Thank for you help!You can run your code now? I have a new problem.I use python3.5 + pytorch1.4 +cuda8.0.I debug it by what it tips.
![image](https://user-images.githubusercontent.com/62084577/76762623-8de7f600-67cc-11ea-87af-f1b51ea674a3.png)
But it happens another pronlem
![image](https://user-images.githubusercontent.com/62084577/76763533-46626980-67ce-11ea-89c6-31aa7f2c39c6.png)

**Comment by zhaizhibo** [comment]
> @naumanbashir70 Thank for you help!You can run your code now? I have a new problem.I use python3.5 + pytorch1.4 +cuda8.0.I debug it by what it tips.
> ![image](https://user-images.githubusercontent.com/62084577/76762623-8de7f600-67cc-11ea-87af-f1b51ea674a3.png)
> But it happens another pronlem
> ![image](https://user-images.githubusercontent.com/62084577/76763533-46626980-67ce-11ea-89c6-31aa7f2c39c6.png)

I have the same problem now. Have you solved that problem?

**Comment by zhaizhibo** [comment]
> @naumanbashir70 Thank for you help!You can run your code now? I have a new problem.I use python3.5 + pytorch1.4 +cuda8.0.I debug it by what it tips.
> ![image](https://user-images.githubusercontent.com/62084577/76762623-8de7f600-67cc-11ea-87af-f1b51ea674a3.png)
> But it happens another pronlem
> ![image](https://user-images.githubusercontent.com/62084577/76763533-46626980-67ce-11ea-89c6-31aa7f2c39c6.png)

how to deal with the problem that the tensor does not match

**Comment by HuangYL-0619** [comment]
@KernLC  I did this job for a long time. Can you pose a picture showing whilch line has this bug?

**Comment by zhaizhibo** [comment]
> @naumanbashir70
> ![image](https://user-images.githubusercontent.com/62084577/76511331-0594e880-648e-11ea-8952-5215ab388d2a.png)

It's the same as this. Line 264. Fuction 'sample()'. The tensor does not match. Which dataset do you use?

**Comment by HuangYL-0619** [comment]
@KernLC  
[lisgan.zip](https://github.com/lijin118/LisGAN/files/5268539/lisgan.zip)
This code is worked.You can try it.Replace your lisgan.py

**Comment by zhaizhibo** [comment]
> @KernLC
> [lisgan.zip](https://github.com/lijin118/LisGAN/files/5268539/lisgan.zip)
> This code is worked.You can try it.Replace your lisgan.py

![image](https://user-images.githubusercontent.com/48124861/94096784-8a0dee00-fe57-11ea-89d4-826ed7246f5d.png)
Thanks for you reply a lot. Do you have this file? **"dataset/ZSL/FLO/res101.mat"**

**Comment by Sakura-rain** [comment]
> @naumanbashir70 I have solved this problem.The name in the file doesn't match the name in the code.You can read "README" and find its name are "att" and "original_att".

I ran into the same problem. Could you please explain in detail where to fix this problem? 
Thank you!

**Comment by Sakura-rain** [comment]
> @yuran1024 @naumanbashir70 I have solved this problem.The name in the file doesn't match the name in the code.You can read "README" and find its name are "att" and "original_att".

I ran into the same problem. Could you please explain in detail where to fix this problem?
Thank you!

**Comment by HuangYL-0619** [comment]
@Sakura-rain  Can you  tell me your e-mail? I can send the worked code to you. Because I run this code a long time ago,I can't  remember the detail.

**Comment by Sakura-rain** [comment]
ldclsfkl@qq.com,this is my e-mail.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "yuran1024"<notifications@github.com&gt;; 
发送时间: 2020年12月21日(星期一) 中午11:03
收件人: "lijin118/LisGAN"<LisGAN@noreply.github.com&gt;; 
抄送: "心有林夕"<851033123@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [lijin118/LisGAN] How to execute LisGAN. (#5)





 
@Sakura-rain  Can you  tell me your e-mail? I can send the worked code to you. Because I run this code a long time ago,I can't  remember the detail.

**Comment by Sakura-rain** [comment]
>@yuran1024@Sakura-rain Can you tell me your e-mail? I can send the worked code to you. Because I run this code a long time ago,I can't remember the detail.

ldclsfkl@qq.com,this is my e-mail.

**Comment by HuangYL-0619** [comment]
@Sakura-rain  You can try it.

**Comment by Sakura-rain** [comment]
>@yuran1024 @Sakura-rain You can try it.

Very grateful for your help!
